src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -172,6 +172,7 @@ load_dh_params(const char *filename) {
   if(filename == NULL) return NULL;
   bio = BIO_new_file(filename, "r");
   if(bio == NULL) return NULL;
+  mtevL(mtev_notice, "Loading DH parameters from %s.\n", filename);
```

Comment:
The goal is: if everything is working normally I should see nothing.  If I'm doing something unique and expensive, perhaps a notice, and if fuck something up I should definitely get an error -- otherwise: debug.